ffi/diplomat/wasm/README.md

```diff
@@ -1,8 +0,0 @@
-# Wasm FFI for ICU4X
```

Comment:
nit: we should probably write a better readme about the various things this folder can do, can be done separately

i.e., mention how to:

 - regen bindings
 - run tests
 - regen and run webpack